src/backlinks/handler.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { badRequest, internalServerError, noContent } from '@adobe/spacecat-shared-http-utils';
+import { hasText, isArray, isObject } from '@adobe/spacecat-shared-utils';
+import {
+  markdown, postSlackMessage, section, uploadSlackFile,
+} from '../support/slack.js';
+
+function convertToCSV(array) {
```

Comment:
could be made more efficient and robust like this:
```suggestion
function convertToCSV(array) {
  if (array.length === 0) {
    return '';
  }

  const headers = Object.keys(array[0]).join(',');
  const rows = array.map(item => {
    return Object.values(item).map(value => {
      if (typeof value === 'object' && value !== null) {
        return `"${JSON.stringify(value)}"`;
      }
      return `"${value}"`;
    }).join(',');
  }).join('\r\n');

  return `${headers}\r\n${rows}\r\n`;
}
```